frontend/src/app/profile/pages/settings/settings.page.ts

```diff
@@ -0,0 +1,184 @@
+      nationality : this.user.nationality
+    };
+    this.presentLoading();
+    this.authController.updateUser(this.userId,newUser).subscribe((res)=>{
+      this.loadingController.dismiss();
+      this.router.navigate(['/profile/settings']);
+    },(err)=>{
+      console.log(err);
+      this.loadingController.dismiss();
+      this.router.navigate(['/profile/settings']);
+    });
+    }
+
+  changeEmail(newEmail: string) {
+    let newUser : User = {
+      name: this.user.name,
+      email: newEmail,
+      profileImage: this.user.profileImage,
+      _id: this.user._id,
+      birth: this.user.birth,
+      city : this.user.city,
+      followers : this.user.followers,
+      following : this.user.following,
+      interests : this.user.interests,
+      nationality : this.user.nationality
+    };
+      this.presentLoading();
+      this.authController.updateUser(this.userId,newUser).subscribe((res)=>{
+        this.loadingController.dismiss();
+        this.router.navigate(['/profile/settings']);
+      },(err)=>{
+        console.log(err);
+        this.loadingController.dismiss();
+        this.router.navigate(['/profile/settings']);
+      });
+    }
+  changeBirth(newBirth : Datetime){
+
+    let newUser : User = {
+      name: this.user.name,
+      email: this.user.email,
+      profileImage: this.user.profileImage,
+      _id: this.user._id,
+      birth: newBirth,
+      city : this.user.city,
+      followers : this.user.followers,
+      following : this.user.following,
+      interests : this.user.interests,
+      nationality : this.user.nationality
+    };
+      this.presentLoading();
+      this.authController.updateUser(this.userId,newUser).subscribe((res)=>{
+        this.loadingController.dismiss();
+        this.router.navigate(['/profile/settings']);
+      },(err)=>{
+        console.log(err);
+        this.loadingController.dismiss();
+        this.router.navigate(['/profile/settings']);
+      });
+
+  }
+  changeNationality(newNationality : string){
+
+    let newUser : User = {
+      name: this.user.name,
+      email: this.user.email,
+      profileImage: this.user.profileImage,
+      _id: this.user._id,
+      birth: this.user.birth,
+      city : this.user.city,
+      followers : this.user.followers,
+      following : this.user.following,
+      interests : this.user.interests,
+      nationality : newNationality
+    };
+    this.presentLoading();
+    this.authController.updateUser(this.userId,newUser).subscribe((res)=>{
+      this.loadingController.dismiss();
+      this.router.navigate(['/profile/settings']);
+    },(err)=>{
+      console.log(err);
+      this.loadingController.dismiss();
+      this.router.navigate(['/profile/settings']);
+    });
+
+  }
+  changeCity(newCity : string){
+
+    let newUser : User = {
+      name: this.user.name,
+      email: this.user.email,
+      profileImage: this.user.profileImage,
+      _id: this.user._id,
+      birth: this.user.birth,
```

Comment:
birth is added along with some other fields to a new encapsulating field in the user data model. Please update the fields accordingly. Good job. 👍